File: tests/go/cases/case_295_shadowed_sink_local_08.go
package escape_tests

var retainedCase295 = []map[string]string{}

func Case295ShadowedSinkLocal08(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "shadowed_sink_local_08",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "shadowed_sink_local_08:" + raw,
	}
	retainedCase := []map[string]string{}
	retainedCase = append(retainedCase, payload)
	// SAFE: local shadow variable only.
	return payload["checksum"]
}

File: tests/nodejs/cases/case_295_shadowed_sink_local_08.js
/**
 * Task case 295: shadowed_sink_local_08 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case295ShadowedSinkLocal08(input) {
  const taskName = 'shadowed_sink_local_08';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const retainedCache = new Map();
  retainedCache.set('tmp', payload);
  // SAFE: local shadow hides global sink symbol.
  return payload.checksum;
}

module.exports = { case295ShadowedSinkLocal08 };
